terasoluna-gfw-functionaltest-web/src/main/webapp/WEB-INF/views/redirect/login.jsp

```diff
@@ -1,7 +1,7 @@
   <h2>Redirect</h2>
   <br>
   <form:form action="${pageContext.request.contextPath}/login" method="post">
-    <c:if test="${param.error}">
+    <c:if test="${!(empty param.error)}">

```

Comment:
- [ ] Is this file for this PR? If this change is for other issue (mis-usage of EL), please create other issue and fix it in other branch.